### PR TITLE
Add support for multiple NOTIFICATION_TELEGRAM_CHAT_ID

### PR DESCRIPTION
**Pull request by gxres042**
With this PR, users can fill in multiple chat ids into NOTIFICATION_TELEGRAM_CHAT_ID like `xxxxx, xxxxx`.

Feature was requested there: https://github.com/orgs/upptime/discussions/956

### CODE REVIEW

File: src/helpers/notifme.ts [hunk 1]
```diff
@@ -331,15 +331,18 @@ export const sendNotification = async (message: string) => {
...
-      await axios.post(
-        `https://api.telegram.org/bot${getSecret("NOTIFICATION_TELEGRAM_BOT_KEY")}/sendMessage`,
-        {
-          parse_mode: "Markdown",
-          disable_web_page_preview: true,
-          chat_id: getSecret("NOTIFICATION_TELEGRAM_CHAT_ID"),
-          text: message.replace(/_/g, '\\_'),
-        }
-      );
+      let chatIds = getSecret("NOTIFICATION_TELEGRAM_BOT_KEY").split(",");
```
Comment (AnandChowdhary) [inline src/helpers/notifme.ts:334]:
```suggestion
      const chatIds = getSecret("NOTIFICATION_TELEGRAM_BOT_KEY").split(",").map((key) => key.trim());
```

File: src/helpers/notifme.ts [hunk 2]
```diff
@@ -331,15 +331,18 @@ export const sendNotification = async (message: string) => {
...
+      for (let chatId of chatIds) {
```
Comment (AnandChowdhary) [inline src/helpers/notifme.ts:335]:
```suggestion
      for (const chatId of chatIds) {
```

**Review by AnandChowdhary — APPROVED** [review]
Thank you for your hard work!

File: src/helpers/notifme.ts [hunk 3]
```diff
@@ -331,15 +331,18 @@ export const sendNotification = async (message: string) => {
...
-      await axios.post(
-        `https://api.telegram.org/bot${getSecret("NOTIFICATION_TELEGRAM_BOT_KEY")}/sendMessage`,
-        {
-          parse_mode: "Markdown",
-          disable_web_page_preview: true,
-          chat_id: getSecret("NOTIFICATION_TELEGRAM_CHAT_ID"),
-          text: message.replace(/_/g, '\\_'),
-        }
-      );
+      const chatIds = getSecret("NOTIFICATION_TELEGRAM_BOT_KEY").split(",").map((key) => key.trim());
```
Comment (AnandChowdhary) [inline src/helpers/notifme.ts:334]:
```suggestion
      const chatIds = getSecret("NOTIFICATION_TELEGRAM_BOT_KEY").split(",");
```

**Comment by AnandChowdhary** [comment]
I think there's something wrong here because there are two variables to deal with, NOTIFICATION_TELEGRAM_BOT_KEY and NOTIFICATION_TELEGRAM_CHAT_ID. Can you please check if we've used the right ones in the logic?

**Comment by gxres042** [comment]
> I think there's something wrong here because there are two variables to deal with, NOTIFICATION_TELEGRAM_BOT_KEY and NOTIFICATION_TELEGRAM_CHAT_ID. Can you please check if we've used the right ones in the logic?

Any ways to use my modified upptime-monitor with my upptime project?

File: src/helpers/notifme.ts [hunk 4]
```diff
@@ -331,15 +331,18 @@ export const sendNotification = async (message: string) => {
...
-      await axios.post(
-        `https://api.telegram.org/bot${getSecret("NOTIFICATION_TELEGRAM_BOT_KEY")}/sendMessage`,
-        {
-          parse_mode: "Markdown",
-          disable_web_page_preview: true,
-          chat_id: getSecret("NOTIFICATION_TELEGRAM_CHAT_ID"),
-          text: message.replace(/_/g, '\\_'),
-        }
-      );
+      const chatIds = getSecret("NOTIFICATION_TELEGRAM_CHAT_ID").split(",");
```
Comment (gxres042) [inline src/helpers/notifme.ts:334]: I wrote `NOTIFICATION_TELEGRAM_BOT_KEY` there by mistake.

It should be `NOTIFICATION_TELEGRAM_CHAT_ID`.

Sorry for that ><

**Review by AnandChowdhary — APPROVED** [review]

**Pull request merged**